addons/sourcemod/scripting/ReservedSlot.sp

```diff
@@ -144,26 +159,37 @@ stock bool KickValidClient(const char[] sName, const char[] sSteam32ID, AdminId
 
 	for(int client = 1; client <= MaxClients; client++)
 	{
-		if(!IsClientInGame(client) || IsFakeClient(client))
+		if(client <= 0 || client > MaxClients || !IsClientInGame(client) || IsFakeClient(client))
```

Comment:
its impossible since we have this for loop `for(int client = 1; client <= MaxClients; client++)`